README.md

```diff
@@ -44,12 +44,31 @@ BOOM runs on macOS and Linux. You'll need:
 
 ## Setup
 
-1. Launch `Valkey`, `MongoDB`, and `Kafka` using docker, using the provided `docker-compose.yaml` file:
+### Environment configuration
+
+BOOM uses environment variables for sensitive configuration like passwords
+and API keys.
+For local development, you can use the defaults in `.env.example`
+by copying it to `.env`:
+
+```sh
+cp .env.example .env
+```
```

Comment:
reminds me a bit of the default config vs just one config debate here. Let's just make sure that the code uses some sensible defaults if some vars are not defined in there? (might already be the case, just asking)